packages/melos/lib/src/command_runner.dart

```diff
@@ -55,4 +63,14 @@ class MelosCommandRunner extends CommandRunner<void> {
     addCommand(PublishCommand());
     addCommand(VersionCommand());
   }
+
+  @override
+  Future<void> runCommand(ArgResults topLevelResults) async {
```

Comment:
Overriding since this is the recommended way to do global flag in command runner: 

![image](https://user-images.githubusercontent.com/5347038/128504576-64bd0920-00c8-49fa-b446-3958d821b4bf.png)